project-base/app/maintenance.html

```diff
@@ -39,6 +37,129 @@
 <body>
 <div class="web">
     <h1>Site Maintenance</h1>
+    <p>
+      <?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
The XML declaration should probably not be included when inlining the svg.

> If the XML declaration is included, it must be situated at the first position of the first line in the XML documentwell-formedness constraint. (Source: https://xmlwriter.net/xml_guide/xml_declaration.shtml)

Inlining SVGs into Twig templates should be avoided, but here it makes sense, as the `maintenance.html` should be without external dependencies.

Furthermore, is it necessary to include the `<p>` tag? It's not a paragraph.

Reply:
Yes, thats true, it is on the top of svg file, but when it is inline included has to be removed. Fixed.